packages/core/src/dialects/abstract/data-types.ts

```diff
@@ -1,3 +1,4 @@
+import type { Falsy } from '@sequelize/utils';
```

Comment:
Falsy is not a type that really makes sense to me. For instance, in this file, it should be replaced with `unknown`

Reply:
Removed now